packages/kotti-ui/source/kotti-field-select/styles.scss

```diff
@@ -29,6 +29,25 @@
 		}
 	}
 
+	&-actions {
+		&:before {
+			position: relative;
+			display: block;
+			height: 1px;
+			margin: 0 16px 0 16px;
+			content: '';
+			background: var(--gray-20);
```

Comment:
i assume all of this is to add a `1px` grey seperator, no?
why the position relative?

Reply:
I don't see an `absolute` wrapper directly but maybe that's the popper's body?